app/models/manageiq/providers/autosde/inventory/parser/storage_manager.rb

```diff
@@ -44,14 +44,22 @@ def physical_storages
     end
   end
 
+  def parse_capabilities(capabilities)
+    caps = JSON.parse(capabilities)
```

Comment:
I'm not sure I understand this - I thought that using jsonb would _automatically_ get capabilities to be a deserialized Hash.